Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml

```diff
@@ -0,0 +1,4175 @@
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "107":
+    id: "107"
+    taskid: 20238e3c-8d69-4727-8061-b63e4bf163f7
+    type: regular
+    task:
+      id: 20238e3c-8d69-4727-8061-b63e4bf163f7
+      version: -1
+      name: Check Whether The Source Host Is an FTP Server
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      key:
+        simple: FTP_Server
+      stringify:
+        simple: "true"
+      value:
+        complex:
+          root: ActiveDirectory.Computers
+          filters:
+          - - operator: match
+              left:
+                value:
+                  simple: ActiveDirectory.Computers.memberOf
+                iscontext: true
+              right:
+                value:
+                  simple: (?i)CN=.*(?:FTP|File).*,
+          accessor: name
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1220,
+          "y": 1680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "109":
+    id: "109"
+    taskid: a9ef2265-0666-4514-865e-23f6b9f94b9a
+    type: regular
+    task:
+      id: a9ef2265-0666-4514-865e-23f6b9f94b9a
+      version: -1
+      name: Check Whether The Source Host Is an SMTP Server
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      key:
+        simple: SMTP_Server
+      stringify:
+        simple: "true"
+      value:
+        complex:
+          root: ActiveDirectory.Computers
+          filters:
+          - - operator: match
+              left:
+                value:
+                  simple: ActiveDirectory.Computers.memberOf
+                iscontext: true
+              right:
+                value:
+                  simple: (?i)CN=.*(?:SMTP|Mail).*,
+          accessor: name
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1620,
+          "y": 1680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "112":
+    id: "112"
+    taskid: 300999f2-2c58-4c9e-8531-e78f8edf4bae
+    type: condition
+    task:
+      id: 300999f2-2c58-4c9e-8531-e78f8edf4bae
+      version: -1
+      name: Calculate Verdict
```

Comment:
SUPAER MASSIVE ONE - can we make it as a subplaybook? cause we won;t be able to debug it due to server limitations...

Reply:
The verdict was calculated at first in a sub-playbook, but Yarden did not want it, so she asked me to minimize the verdict calculation into a single task.